spec/lib/agris/client/grain/purchase_contracts_spec.rb

```diff
@@ -38,8 +39,10 @@
         result = client.purchase_contract(contract_location, contract_number)
 
         expect(result.remarks.length).to eq(1)
-        expect(result.remarks[0]).to have_key('remark')
-        expect(result.remarks[0]['remark']['value']).to eq(remark)
+        expect(result.remarks.first).to have_key('remark')
+        expect(result.remarks.first['remark'].length).to eq 2
+        expect(result.remarks.first['remark'].first['value']).to eq(remark4)
+        expect(result.remarks.first['remark'].last['value']).to eq(remark14)
```

Comment:
How do I know this is remark 14 and not 13? Don't you need to parse out the number from the remark?

The `number="04"` part of the XML and maybe output of something like

```
remarks = {
  '01': "The 1st remark",
  '12': "The 12th remark",
  '14': "The 14th remark"
}
```

Then you can always know you are getting remark 14 via
```
remark['14']
```

Reply:
Currently, the remarks are returning with this structure: 
```
 [{ 'remark' => [{ 'type' => 'Contract',
                                    'number' => '04',
                                    'label' => 'RMK#04',
                                    'code' => '',
                                    'value' => 'Test Remark #4' },
                                  { 'type' => 'Contract',
                                    'number' => '14',
                                    'label' => 'RMK#14',
                                    'code' => '',
                                    'value' => 'Test remark #14' }] }]
```

Reply:
Oh I see, so you could map those in Otis/Contracto. Got it

:+1: